tools/include/edm4hep2json.hxx

```diff
@@ -28,14 +32,21 @@
 #include <iostream>
 #include <vector>
 
-template <typename PodioStoreT>
-nlohmann::json processEvent(PodioStoreT& store, std::vector<std::string>& collList, bool verboser, unsigned eventNum) {
-  const podio::CollectionBase* coll;
+template <typename StoreT = podio::EventStore>
+nlohmann::json processEvent(StoreT& store, std::vector<std::string>& collList, bool verboser, unsigned eventNum,
```

Comment:
Do we want to keep this with legacy support? Otherwise this could be an untemplated function that takes a `const podio::Frame&` as its first argument. That would potentially also fix the CI build errors for some of the clang builds.

Reply:
Probaly not. Removed in the next commit.